src/api/tx_submit.rs

```diff
@@ -30,7 +30,15 @@ pub async fn route(
     .await
     .expect("submit_transaction panic!")?;
 
-    Ok(Json(response))
+    let mut response_headers = HeaderMap::new();
+
+    // insert a static header value. Using unwrap is safe here because the literal is known to be a valid header.
+    response_headers.insert(
+        "blockfrost-platform-response",
+        "HeaderValue".parse().unwrap(),
```

Comment:
It’s only a static value. :upside_down_face:  I reopened the original issue:
* #230

@vladimirvolek said he’ll fix it.